.github/workflows/e2e.yaml

```diff
@@ -23,6 +23,10 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      # For network flakiness. See https://github.com/vercel/next.js/pull/28264
+      - name: Tune linux network
+        run: sudo ethtool -K eth0 tx off rx off
```

Comment:
If this works, we should add to add workflows, setting a condition to run in case of Linux only.